geometry_msgs/Point[] impact_points
float32 impact_times